questions/qa-backwards-deletion.en.html

```diff
@@ -358,36 +349,16 @@ <h4>Try it in your browser</h4>
   
 </section>
 
-<!--
 <section>
-	<div style="width:80%; background: #dfdfdf; border: 1px solid black">
-<h4>Try it yourself!</h4>
-
-<p>Each of the examples given above can be used to explore cursoring and deletion on your browser.</p>
-<form action="qa-backwards-deletion.en.html" method="get" id="form" name="form">
-
-<p>
-<select id="deletionExamples" name="deletionExamples" onchange="updateFromSelect()" onload="updateFromSelect()">
-	<option id="value" name="hindi" lang="hi">&#x92f;&#x942;&#x928;&#x93f;&#x915;&#x94b;&#x921;</option>
-	<option id="value" name="tamil" lang="ta">&#xb95;&#xbcb;</option>
-	<option id="value" name="thai" lang="th">&#xe04;&#xe37;&#xe2d;&#xe30;&#xe44&#xe23;</option>
-	<option id="value" name="korean" lang="ko">&#xac01; &#x1100;&#x1161;&#x11a8;</option>
-	<option id="value" name="latin" lang="en">&#x1fa; &#x41;&#x30a;&#x301;</option>
-	<option id="value" name="emoji" lang="en">&#x1f468;&#x200d;&#x1f469;&#x200d;&#x1f467;&#x200d;&#x1f467;</option>
-	<option id="value" name="conjunct" lang="hi">&#x915;&#x94d;&#x937;&#x93f;</option>
-</select>
-</p>
-
-<p id="exampleExplainer"></p>
-
-<p><input type="text" id="example" name="example" style="font-size:24pt">Explanation of example</p>
-
-<input type=submit>
-
-
-</form>
+	<h3>Vertical text</h3>
+	
+      <p>Vertical text works in a similar way to horizontal text with regard to text selection, cursoring, deletion, and backspacing. As you might suspect, cursor keys on a keyboard in general move the cursor in the direction of the arrow on the key. This usually means that, in vertical text, "up" is generally preceding the current cursor position logically, while "down" is generally following, while "left" and "right" navigate between lines of text.</p>
+  
+      <!-- Chinese text is from the zh-Hans "What is Unicode?" page -->
+	  <h4>Try it in your browser</h4>
```

Comment:
Since [the browsers do not support it universally yet](https://github.com/w3c/clreq/issues/247), we should probably add a note to explain that this might not be rendered correctly in Chrome and Safari.

Reply:
Oh, yes, I forgot about that. Fixed.